chat-plugins/room-faqs.js

```diff
@@ -91,7 +91,8 @@ exports.commands = {
 		if (!this.runBroadcast()) return;
 		this.sendReplyBox(Chat.formatText(roomFaqs[room.id][topic], true));
 		if (!this.broadcasting && user.can('declare', null, room)) {
-			let extra = `<code>/addfaq ${topic}, ${Chat.escapeHTML(roomFaqs[room.id][topic])}</code>`;
+			let src = Chat.escapeHTML(roomFaqs[room.id][topic]).replace(/\n/g, '<br />');
+			let extra = `<code>/addfaq ${topic}, ${src}</code>`;
 			let aliases = Object.keys(roomFaqs[room.id]).filter(val => getAlias(room.id, val) === topic);
 			if (aliases.length) {
 				extra += `<br/><br/>Aliases: ${Object.keys(roomFaqs[room.id]).filter(val => getAlias(room.id, val) === topic).join(', ')}`;
```

Comment:
`<br />`

Reply:
Also is there any reason you're re-running the query instead of outputting `aliases.join(...)`?

Reply:
that was an oversight for sure